packages/synchro-charts/src/components/sc-dial/sc-dial.tsx

```diff
@@ -0,0 +1,62 @@
+import { Component, h, Prop } from '@stencil/core';
+import { DataPoint, DataStream, MessageOverrides, StreamAssociation, ViewPortConfig } from '../../utils/dataTypes';
+import { Annotations, DialConfig, Threshold } from '../charts/common/types';
+import { breachedThreshold } from '../charts/common/annotations/breachedThreshold';
+import { isMinimalStaticViewport } from '../../utils/predicates';
+import { getThresholds } from '../charts/common/annotations/utils';
+import { DialSizeConfig } from './type';
+
+@Component({
+  tag: 'sc-dial',
+  styleUrl: 'sc-dial.css',
+  shadow: false,
+})
+export class ScDial implements DialConfig {
```

Comment:
we don't need a `DialConfig` interface here. `DialConfig` is not a 'common chart type' and I believe can be removed entirely from the code base.

Stencil automatically generates typescript types based on the component, so if there is

Reply:
Done --- Removed `DialConfig`  interface